### DISCUSSION TITLE
마크다운 버튼 호버시 색 변화

### DISCUSSION THREAD

**Discussion by ahnichan** [The-Dark-Nights/front-end#38] [General]
에디터에서 제공하는 버튼들은 마우스 호버시 색깔이 바뀌는 효과가 있는데 제가 커스텀한 버튼은 색변화가 안됩니다.
다른 방법을 찾아봐야할 것 같아용

**Comment by max0901** [comment]
onMouseOver 이벤트를 사용하면됩니다 상태 값을 기본을 false로 주고 올라갔을떄 변하는 css를 하나 만들어놓고 마우스올라갈때마다 true변하는 값으로 바꾸면서 태그에는 클래스에 삼항연산자를써서 false일떄 기본 클래스 true일떄 변하는css클래스로 두면 가능합니다
제가 다른곳에 마우스 오버를 만들어놨으니 한번 보셧으면 합니다

**Reply by ahnichan** [reply]
시도해보겠습니다!

**Comment by ahnichan** [comment]
버튼에 삽입된 이미지 형식을 svg로 바꾼 뒤에 호버 효과를 주어 해결했습니다.
일반 이미지는 색 변화가 이벤트마다 바뀌지 않았는데 svg 형식은 잘 바뀌더라구요
